### PR TITLE
DLS-1389 Upgrade Play2.6 changes

### PR DESCRIPTION
**Pull request by rajanimohan22**
DLS-1389 Upgrade Play2.6 changes

### CODE REVIEW

File: app/connectors/DesConnector.scala [hunk 1]
```diff
@@ -53,8 +53,9 @@ case object DesGetUnexpectedResponse extends DesGetResponse
 class DesConnector @Inject()(val runModeConfiguration: Configuration,
                              metrics: ApplicationMetrics,
                              http: HttpClient,
-                             auditConnector: AuditConnector)
-  extends RawResponseReads with ServicesConfig {
+                             auditConnector: AuditConnector,
+                             val servicesConfig: ServicesConfig)
+  extends RawResponseReads {
 
   protected def mode: Mode = Play.current.mode
```
Comment (calinursaciuc) [inline app/connectors/DesConnector.scala]: check if this is used anywhere...if not delete it

File: app/models/GmpCalculationResponse.scala [hunk 2]
```diff
@@ -16,8 +16,9 @@
 
 package models
 
-import org.joda.time.LocalDate
-import play.api.libs.json.Json
+import java.time.LocalDate
```
Comment (calinursaciuc) [inline app/models/GmpCalculationResponse.scala]: Make sure joda and java Local Date use same format for backward compatibility

File: conf/application.conf [hunk 3]
```diff
@@ -18,7 +18,8 @@ include "backend.conf"
 appName = gmp
 
 play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
-play.http.requestHandler ="play.api.http.GlobalSettingsHttpRequestHandler"
+#play.http.requestHandler ="play.api.http.GlobalSettingsHttpRequestHandler"
```
Comment (calinursaciuc) [inline conf/application.conf]: Delete the unused one

File: app/models/GmpCalculationResponse.scala (hunk 2 above)
Comment (rajanimohan22) [reply app/models/GmpCalculationResponse.scala]: Gone through all the code places where this DateTime is used. Execution is fine. But if any uncovered areas are there, they might get exposed during automation testing.
No Action taken for  this.

File: app/connectors/DesConnector.scala (hunk 1 above)
Comment (rajanimohan22) [reply app/connectors/DesConnector.scala]: Deleted.

File: conf/application.conf (hunk 3 above)
Comment (rajanimohan22) [reply conf/application.conf]: Done

**Comment by calinursaciuc** [comment]
Will merge after acceptance tests pass

**Pull request merged**